doxia-site-renderer/src/test/java/org/apache/maven/doxia/siterenderer/XhtmlEntityResolver.java

```diff
@@ -1,83 +0,0 @@
-package org.apache.maven.doxia.siterenderer;
```

Comment:
What if someone else is using an extension to the XhtmlSink and still requires this? Maybe mark for deletion in 2.0/3.0?

Reply:
My understanding of how maven-doxia-sitetools works is that it takes many possible inputs (markdown, etc etc) and generates just one kind of output - previously xhtml4.

After this change, we take many possible inputs and generate just one kind of output - xhtml5.

Anyone needing to depend on XhtmlSink would need to depend on an older jar.

I was anticipating the switch of output from xhtml4 to xhtml5 to be big enough to require a bump to v2.0, but I leave that decision to you.